### ISSUE TITLE
How can I get the target torque and current for UR10 robot?

### ISSUE THREAD

**Issue by MohamedOmar2014** [UniversalRobots/Universal_Robots_ROS_Driver#657]
(ROS neotic+ubuntu 20.04)
I'm trying to get the target torque and current for each joint of the robot. I added target_current and target_moment to rtde_output_recipe.txt but I don't know how can I access it through /joint_states topic or any other way. Any help??

I did that based on the following:
 * To make use of #35 you can try to checkout that branch and add the joint currents or `target_moment` fields to the output recipe file. This is currently a feature in development, so sorry that there isn't a ton of documentation there, yet.
@fmauch

**Comment by fmauch** [comment]
Yes, that would have been the suggestion I would have done. #35 comes closest to what you want to achieve. As you can see in the discussion there, this has never settled to an agreement and never was followed up.

**Comment by urfeex** [comment]
Since this has been lurking around for ages now since #35 has been created, there is a solution for ROS 2 now:

https://github.com/UniversalRobots/RTDE_ROS2_Publisher contains a ROS 2 node publishing any RTDE output directly to a ROS topic. For ROS 1 this will not be implemented anymore, hence I'm closing this issue.
